qa/rpc-tests/wallet_shieldcoinbase.py

```diff
@@ -83,14 +83,14 @@ def run_test (self):
             self.nodes[0].z_shieldcoinbase("*", myzaddr, Decimal('21000000.00000001'))
         except JSONRPCException as e:
             errorString = e.error['message']
-        assert_equal("Amount out of range" in errorString, True)
+        assert_equal(errorString, "Amount out of range")
 
         # Shielding will fail because fee is larger than sum of utxos
         try:
             self.nodes[0].z_shieldcoinbase("*", myzaddr, 999)
         except JSONRPCException as e:
             errorString = e.error['message']
-        assert_equal("Insufficient coinbase funds" in errorString, True)
+        assert_equal(errorString, "Insufficient funds: have 50.00, need 999.00.")
```

Comment:
This error message should probably make clear whether the units are ZAT or ZEC

Reply:
I added a `DisplayAmount` in a different branch … somewhere. Anyway, it’s basically `FormatAmount`, but prefixes with `CURRENCY_UNIT`. Later, I’d like it to also have thousands separators in it.

Since this is an existing error message, I’d like to do that in a separate PR, too, and do a `s/FormatAmount/DisplayAmount/g` more broadly.

Reply:
Created https://github.com/zcash/zcash/issues/6570 to track this.